core/src/main/java/org/apache/accumulo/fate/Fate.java

```diff
@@ -81,6 +81,9 @@ public void run() {
 
               if (deferTime == 0) {
                 prevOp = op;
+                if (status.equals(TStatus.SUBMITTED)) {
+                  store.setStatus(tid, TStatus.IN_PROGRESS);
+                }
```

Comment:
enums should always be compared with `==` and not `.equals`

```suggestion
                if (status == TStatus.SUBMITTED) {
                  store.setStatus(tid, TStatus.IN_PROGRESS);
                }
```